cockroachcloud/quickstart.md

```diff
@@ -24,22 +28,34 @@ The sample code used in this tutorial is located in the [`quickstart-code-sample
   <button class="filter-button" data-scope="java">Java</button>
 </div>
 
+<section class="filter-content" markdown="1" data-scope="serverless">
+
 ## Step 1. Create a free cluster
 
 {% include cockroachcloud/quickstart/create-a-free-cluster.md %}
 
-## Step 2. Create a SQL user
+### Create a SQL user
```

Comment:
Is there a reason why this isn't a step? It looks a little odd now, having this "task" as a floating level 3 header.

Reply:
Yes. Since we hard-code the step numbers in the headers, the number of steps for Serverless was one more than for demo, so the shared headers wouldn't match.

I could pull every common step's content out into an include, but that makes it harder to edit.

Reply:
Since Jesse suggested consolidating the demo steps, I've conditionalized all the step headings, so I reverted this back to its own step.